### PR TITLE
Kritis integrations skeleton

### LINKED ISSUE #19: Integration tests for Kritis.
Need to design and implement integration testing for Kritis.
Deploying all objects i.e. CRDs, admission controller via helm.
1. Deploy a container which violates image security policy 
2. Deploy a container which complies image security policy.

### PR DESCRIPTION
**Pull request by aaron-prindle**
Fixes #19

### CODE REVIEW

File: hack/build_docs.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```
Comment (dlorenc) [inline hack/build_docs.sh]: do we need this?

File: deploy/kritis/pod.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kritis
+spec:
+  containers:
+  - name: kritis
+    image: gcr.io/kritis-int-test/kritis
+    command: ["/bin/bash", "-c", "--" ]
```
Comment (dlorenc) [inline deploy/kritis/pod.yaml]: what's this yaml for?

File: deploy/kritis/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/gcp-runtimes/ubuntu_16_0_4 as runtime_deps
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends --no-install-suggests -y \
+        python-dev
+
+ENV KUBECTL_VERSION v1.10.0
```
Comment (dlorenc) [inline deploy/kritis/Dockerfile]: Why do we need all of these?

File: deploy/cloudbuild.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# using default substitutions, provided by Google Container Builder
```
Comment (dlorenc) [inline deploy/cloudbuild.yaml]: remove

File: hack/build_docs.sh (hunk 1 above)
Comment (aaron-prindle) [reply hack/build_docs.sh]: no, deleted this and all other unused files added in hack/

File: deploy/kritis/pod.yaml (hunk 2 above)
Comment (aaron-prindle) [reply deploy/kritis/pod.yaml]: deleted

File: deploy/cloudbuild.yaml (hunk 4 above)
Comment (aaron-prindle) [reply deploy/cloudbuild.yaml]: deleted

File: deploy/kritis/Dockerfile (hunk 3 above)
Comment (aaron-prindle) [reply deploy/kritis/Dockerfile]: removed unneeded packages.  We need kubectl, cloud_sdk in the current test iteration and helm and docker-credential-helper for future tests

File: integration/run_test.go [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+			t.Fatalf("testing error: %v", err)
+		}
+	}
+}
+
+func TestCRDs(t *testing.T) {
+	_, deleteNs := setupNamespace(t)
+	defer deleteNs()
+
+	// CRDs themselves are non-namespaced so we have to delete them each run
```
Comment (dlorenc) [inline integration/run_test.go]: Should you add another call to this, defered?

**Comment by tejal29** [comment]
I think this is going to conflict with #46. PTAL.

File: integration/run_test.go (hunk 5 above)
Comment (aaron-prindle) [reply integration/run_test.go]: yea, added

**Comment by aaron-prindle** [comment]
updated this pr to install kritis via helm after #46

**Pull request merged**